### PR TITLE
Compatibility fix (build example using latest Rust toolchain)

### PR DESCRIPTION
**Pull request by bucanero**
This PR adds a dependency so the FT example can be built using the latest Rust toolchain (2021-06-17)

### CODE REVIEW

**Review by thisisjoshford — APPROVED** [review]
LGTM! Let me ask one of our senior devs to double check this before we merge @bucanero.

File: Cargo.toml
Comment (austinabell) [inline Cargo.toml:10]:
```suggestion
```

**Review by austinabell — COMMENTED** [review]
So I just checked, this macro sanity issue was in fact fixed with the 3.1.1 release, but you just need to update the versions in the two crates that reference it, not add the new import. Seems like this crate just acts as a wrapper to sim test with the two contracts, so it doesn't need contract-standards.

File: Cargo.toml
Comment (austinabell) [inline Cargo.toml:9]: Small nit but do you mind switching to this version in the fungible token contract too? Just in case someone copies just that part of the example

Comment (bucanero) [reply Cargo.toml:9]: hi @austinabell , thanks for the review

I have updated the version and committed `Cargo.lock`. Let me know if I missed something.

**Review by austinabell — APPROVED** [review]
LGTM! This will be sufficient, but the version of `near-contract-standards` could be bumped inside the fungible token example to be safe. Doesn't really matter though because even if someone copied that directory out, the version would default to above the broken one anyway.

Not sure what senior dev Josh is waiting on but here's my approval anyway!

File: Cargo.toml
Comment (bucanero) [reply Cargo.toml:9]: Also updated `./ft/Cargo.toml` ✅

**Pull request merged**
